Cubical/Structures/Macro.agda

```diff
@@ -31,6 +32,8 @@ data Desc (ℓ : Level) : Typeω where
   param : ∀ {ℓ'} → (A : Type ℓ') → Desc ℓ  → Desc ℓ
   -- structure S parameterized by variable argument: X ↦ (X → S X)
   recvar : Desc ℓ  → Desc ℓ
+  -- Maybe on a structure S: X ↦ Maybe (S X)
```

Comment:
Isn't maybe a functorial action?

Reply:
On its own, yes, but not necessarily in combination with other things. For example X |-> Maybe (X -> X) isn't functorial.